### PR TITLE
Update CSS build system requirements

### PR DESCRIPTION
**Pull request by gsnedders**
Fixes https://github.com/w3c/web-platform-tests/issues/7200#issuecomment-367259846.

This cannot be merged until https://lists.w3.org/Archives/Public/public-css-testsuite/2018Feb/0001.html has landed in w3ctestlib and then this should be amended so that the w3ctestlib submodule is updated before the requirements update.

cc/ @jgraham @foolip

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-02-21 15:33:14
Finished: 2018-02-21 15:43:05


<h2>Failing Jobs</h2>
<ul>

<li>build_css</li>

<li>MicrosoftEdge:16.16299</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/24655)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/344365427)

File: css/build-css-testsuites.sh [hunk 1]
```diff
@@ -2,11 +2,11 @@
 set -ex
 
 SCRIPT_DIR=$(dirname $(readlink -f "$0"))
```
Comment (gsnedders) [inline css/build-css-testsuites.sh]: This is broken because somehow on CI "$0" is actually `tools/ci/run.sh` o_O

**Comment by Ms2ger** [comment]
Causes issues like `./tools/ci/run.sh: line 9: tools/ci/ci_lint.sh: Permission denied`

**Comment by foolip** [comment]
What's w3ctestlib and why is it maintained with patches on a mailing list? :)

**Comment by plinss** [comment]
w3ctestlib is part of the csswg test suite build system (which predates wpt by many years), it's still in use by the csswg test harness.

The primary repo for w3ctestlib is on the csswg's mercurial server but there is a mirror on github: https://github.com/w3c/csswg-w3ctestlib

Changes to w3ctestlib do need to get upstreamed to the mercurial repo for the csswg infrastructure to get them, hence the patch sent to me, but a PR to the github repo also would have worked (and in a pinch, would have unblocked wpt).

File: css/build-css-testsuites.sh (hunk 1 above)
Comment (gsnedders) [reply css/build-css-testsuites.sh]: Should be fixed now?

**Comment by gsnedders** [comment]
OK, this should be ready to land now. See https://travis-ci.org/w3c/web-platform-tests/builds/346437609 for all test jobs being run.

**Review by Ms2ger — APPROVED** [review]

**Pull request merged**